frontend/src/assets/docs/profiles.en.md

```diff
@@ -20,16 +20,16 @@ If I am neither holding a mouse nor my camera, I grab my Vic Firth Sticks and ta
 
 ## Florian
 
-The first contact with Computers I had in third grade. At that time I got my first own computer.
+Kotlin enthusiast, caffeine-junkie and sailor
 
-Thanks to the fact that I could use it in school, because of my physical disability, I rapidly spent a lot of time with it.
+My main focus in Outcobra is on the application logic.
+This means that I am responsible for the fact that the data you are typing is stored in the correct format, and will be available to you in the future on every device.
 
-Five years later I began to teach programming myself.
+Designing software never gets boring. If these work, at the end of the day, the feeling is indescribable.
+For this reason, and to keep improving myself, I spend a lot of my spare time with software development.
+On my desk, besides the mouse and the keyboard, a cup of coffee can not be missed.
```

Comment:
Besides the mouse and the keyboard, a cup of coffee can't be missed on my desk.

Sounds better IMO